### ISSUE TITLE
Suggestions for GUI improvements

### ISSUE THREAD

**Issue by jcwinkler** [textext/textext#16]
(Originally posted by @sizmailov in pull request #14):
 
1. Don't show or disable "Reset" button for newly created nodes, since it does nothing except generating non-sense error messages when Ok button gets pressed (at least in GTK)
 2. Also "Global" label is misleading, since it actually sets scale to last used. Not sure how to name such action, "Set as previous" sounds right, but it's a bit wordy. Again, don't show button if there is no "previous" node.

**Comment by jcwinkler** [comment]
![grafik](https://user-images.githubusercontent.com/11866252/39759364-c3e37948-52d2-11e8-8eb1-5ee8352a981a.png)

Changes
- Scaling and Alignment in two different boxes
- Selection of TeX converter

What remains open:
- Naming of "Reset" and "Global":
   - `Reset` actually means "Reset to value used for creating this node"
   - `Global` actually means "Use scale factor of previously edited node"
   - Any short word for describing this? 
   - The tooltips do it but they are not available in the TK version and, personally, I am not willing to implement them via additional classes like Pmw ...
- Alternative title of groupbox "Alignment": "Alignment to old node" or something like this?

Any comments?

**Comment by sizmailov** [comment]
What are actual use cases for `Global` button? 

I can imaging two potential scenarios:
1) Create several nodes in a row with a given scale factor
    Now scale factor gets initialized with previous value, so what button `Global` actually does is *resets* value to original. Let's remove `Global` button for fresh nodes and let `Reset` button do reset. 
2) Adjust scale of current node to match previous node
    This allows one to avoid copy-paste scale factors across nodes. Actually it's scale factors history of length 1. We can store several (say, up to 5) previously used values and allow user to pick one of them, see snapshot below. 

**Label names**
Now low precision in button labels is misleading, I suggest to remove it, i.e. "Reset (1.0)" would be just "Reset". 

- `Reset` actually means "Reset to value which was when window appeared", so just "Reset" is fine. 
- `Global` "As previous" lacks verb, but intent is still clear

The final window may look like this:
![image](https://user-images.githubusercontent.com/2975991/39785810-33fc84d8-5326-11e8-9fbd-086d1a1da293.png)

**Comment by jcwinkler** [comment]
"As previous" is a very good suggestion! Thanks!

**Comment by jcwinkler** [comment]
Yes, good idea!
